README.md

```diff
@@ -32,6 +32,7 @@ and any personal details or requests are lost in the handoff.
 - [ ] Inspect that the windshield isn't installed shifted (the gaps between the glass and A pillar on either side
 should be approximately equal). An improperly installed windshield causes significant wind noise at highway speeds.
 - [ ] Check for uneven/wide panel gaps and that panels are flush to car
+- [ ] Check that the trim along the bottom of the sides of the vehicle are flush with the bottom of the doors
```

Comment:
Any chance to have photos showcasing this problem? If so, it would be great to add links.

Reply:
https://twitter.com/StratusBehr/status/1341081320148111364/photo/1